### ISSUE TITLE
FR: enhance & move output

### ISSUE THREAD

**Issue by geziefer** [UoA-eResearch/hololens_facial_recognition#4]
After making it work, I think others may stumble upon the same points as me, that 1st) there is no meaningful output when the REST call doesn't work for some reason. And 2nd) it would be great, if the output follows my view instead of staying at a fix position.
The 1st is propably simple, but I have yet no idea if the 2nd is.
And an additional basic question: Do I always have to go from the Unity project, or can I stick to the Visual C# solution once it's exported for the 1st time? (by the way, it may make sense to add to the readme, what you told me to use as base tutorial for it).

**Comment by neon-ninja** [comment]
The problem with 2) is that the text/rectangle is positioned in space next to the face it's associated with. If we move that, in the case of multiple faces, it may become unclear which text corresponds to which face. Perhaps leaving the rectangle fixed and drawing a line from the rectangle to the text might help with that.

For what purpose? Launching the app? Once it's deployed once you don't need a computer anymore, you can launch it from the hololens itself

**Comment by geziefer** [comment]
I will experiment with the project myself and see what one can do with it, thanks anyway with the good start you gave!
Concerning IDE question, I meant, whether I have to work in Unity and continuously have to export into Visual Studio solution to deploy on HoloLens or if I can, once it is built into an Visual C# project, stay within Visual Studio and continue developing there.

**Comment by neon-ninja** [comment]
I'm unsure if the unity build process modifies the source files in any way. Can you run a find . | grep in the built solution App folder for GazeGestureManager ? I don't currently have access to a built version of the project

**Comment by geziefer** [comment]
No, it still resides in  ../Assets/Scripts/GazeGestureManager.cs

**Comment by neon-ninja** [comment]
then I think editing that should still affect the vs deploy
